src/lambdas/get-app-theme-color.lambda.ts

```diff
@@ -57,17 +85,31 @@ export const getButtonColorLambda = (
   if (!theme || theme === AppTheme.Basic) return undefined
 
   // by default halloween as only two options for now:
+  // if (disabled) {
+  //   return {
+  //     color: Halloween.three, // it will be the font color
+  //     borderColor: Halloween.one, // it will be the border color
+  //     backgroundColor: Halloween.one,
+  //   }
+  // }
+
```

Comment:
Don't comment it out!

Reply:
Handled in [8ef685e](https://github.com/ajktown/ConsistencyGPT/pull/74/commits/8ef685e54a10fbb3a6fff4f9964b248e91c083eb)